src/main/java/eu/deltasource/internship/hotelmanagementsystem/hotel/service/domain/commodities/Booking.java

```diff
@@ -1,39 +1,74 @@
-package commodities;
+package eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities;
 
-import project.Room;
 
 import java.time.LocalDate;
 
+/**
+ * Class booking has 3 private members
```

Comment:
Not very helpful javadoc, You must maintain this as well, if you add a variable this will change...
It would be better to have some description of what the class does. f.e. "This represents a Booking for a room, which is created by a guest."